### DISCUSSION TITLE
How to set a fan speed ceiling or more than a simple offset for sync?

### DISCUSSION THREAD

**Discussion by goathens** [Rem0o/FanControl.Releases#764] [Q&A] [answered]
I'm loving this software, it's much more featureful than my BIOS and more reliable than other freeware/open-source options (and even some for-pay options too).

I'm unable to control my GPU fan curve with FanControl, so I've resolved to make a "sync" curve between my case fans and the gpu fan. The goal is for the case fans to kick in when the gpu fans fire up, to "feed fresh air".

The issue is that I don't want my case fans to spin at the same %speed as the gpu.  The gpu fan curve is all over the place, and the case fans are much louder when they approach maximum.  The "sync" fan curve has a simple offset value which is limited to -30% and isn't really what I need.

I could see a few features that could make this work:
- Simplest option: put a maximum speed limit onto my case fans, but I don't see how to do that with the software as it is now.
- Add different options to the "sync" fan curves. Not only an "offset" (subtraction), but maybe an arbitrary mathematic adjustment "50% of the synced fan's speed".
- Add a fan's RPM/% output as one of the available "Sensors" for all of the other fan curves.  A linear curve where 20% gpu fan == 20% case fan, but 80% gpu fan == 55% case fan.

**Comment by Rem0o** [comment, marked as answer]
You can use an offset on the fan itself: 
![image](https://user-images.githubusercontent.com/14118956/167952513-8d15b42e-a3ec-499a-ab71-e0736bd3954d.png)

Put -20 to limit it to 80%, and voila.

**Reply by goathens** [reply]
Excellent, thank you. That's way better than my solution here:

Use a "Flat" curve set to the ceiling, and then use a "Mix" curve using the minimum between the "Flat" curve and the "Sync" curve.
![image](https://user-images.githubusercontent.com/66317503/167962244-3f871fc6-11e1-4451-a338-4eec678a763f.png)

**Reply by Rem0o** [reply]
That works too!
